examples/python/s2ag-datasets.py

```diff
@@ -0,0 +1,60 @@
+"""
+Sample usage of Semantic Scholar Academic Graph Datasets API
+https://api.semanticscholar.org/api-docs/datasets
+"""
+import requests
+import urllib
+import json
+import os
+
+# Get info about the latest release
+latest_release = requests.get("http://api.semanticscholar.org/datasets/v1/release/latest").json()
+print(latest_release['README'])
+print(latest_release['release_id'])
+
+# Get info about past releases
+dataset_ids = requests.get("http://api.semanticscholar.org/datasets/v1/release").json()
+earliest_release = requests.get(f"http://api.semanticscholar.org/datasets/v1/release/{dataset_ids[0]}").json()
+
+# Print names of datasets in the release
+print("\n".join(d['name'] for d in latest_release['datasets']))
+
+# Print README for one of the datasets
+print(latest_release['datasets'][2]['README'])
+
+# Get info about the papers dataset
+papers = requests.get("http://api.semanticscholar.org/datasets/v1/release/latest/dataset/papers",headers={'x-api-key':os.getenv("S2_API_KEY")}).json()
+
+# Download the dataset
+urllib.request.urlretrieve(papers['files'][0], "papers-part0.jsonl.gz")
+
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # 
+# Explore datasets
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+## Papers
+papers = [json.loads(l) for l in open("papers/papers-sample.jsonl","r").readlines()]
+
+## Citations
+citations = [json.loads(l) for l in open("citations/citations-sample.jsonl","r").readlines()]
```

Comment:
Should we also download part0 of the citations dataset and unzip it for this line to execute without an error? Same for the datasets below.